pkg/resources/admission_webhooks.go

```diff
@@ -0,0 +1,149 @@
+package resources
+
+import (
+	"fmt"
+
+	"github.com/nukleros/operator-builder-tools/pkg/controller/workload"
+	admissionv1 "k8s.io/api/admissionregistration/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	MutatingWebhookConfigurationKind    = "MutatingWebhookConfiguration"
+	MutatingWebhookConfigurationVersion = "admissionregistration.k8s.io/v1"
+
+	ValidatingWebhookConfigurationKind    = "ValidatingWebhookConfiguration"
+	ValidatingWebhookConfigurationVersion = "admissionregistration.k8s.io/v1"
+)
+
+type MutatingWebhookConfigurationResource struct {
```

Comment:
Can we add a doc comment here?

Reply:
Fixed in [46cbf84](https://github.com/nukleros/operator-builder-tools/pull/5/commits/46cbf8426e1841f3bd28610071247ea56c6ab8c8)